packages/manager/modules/nutanix/src/dashboard/nodes/node/interventions/template.html

```diff
@@ -0,0 +1,4 @@
+<server-interventions
+    service-name="{{ $ctrl.node.name }}"
+    on-error="$ctrl.handleError(error)"
+></server-interventions>
```

Comment:
I'm wondering if we could use this component directly in routing and avoid a view + controller in this case

Reply:
I prefer to use routes only for pure resolver methods/variables. Controller for other kind of methods. If I directly use the component inside the route, I need to define the handleError method inside the route which I would like to avoid. Also having a controller provides flexibility to add other features easily like tracking etc.